pkgs/development/python-modules/ydata-profiling/default.nix

```diff
@@ -0,0 +1,97 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, pytestCheckHook
+, fastparquet
+, htmlmin
+, imagehash
+, jinja2
+, matplotlib
+, multimethod
+, numpy
+, pandas
+, phik
+, pydantic
+, pyyaml
+, requests
+, scipy
+, seaborn
+, statsmodels
+, tqdm
+, typeguard
+, visions
+}:
+
+buildPythonPackage rec {
+  pname = "ydata-profiling";
+  version = "4.1.2";
+  disabled = pythonOlder "3.7";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "ydataai";
+    repo = pname;
```

Comment:
```suggestion
    repo = "ydata-profiling";
```

Reply:
See my comment on the other package added in this commit.